config/recipes/beats/filebeat_openshift.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: beat.k8s.elastic.co/v1beta1
```

Comment:
This file differs with `filebeat_autodiscover.yaml` file only by:
1. version (we should fix it in all the other recipes separately)
2. namespace (all other recipes assume default namespace, maybe we can do here as well?)
3. `privileged: true`

I'm not sure if it's worth to introduce a new file just to flip a `privileged` to `true`. Couldn't we point users to reuse the already existing file?

Reply:
Instead of 2 separate files - I've created a new file `openshift_monitoring.yaml` including both filebeat and metricbeat to monitor openshift within a single manifest